quipucords/fingerprinter/tests_fingerprint_task.py

```diff
@@ -1138,6 +1138,7 @@ def test_process_details_report_success(self):
                 'display_name':
```

Comment:
The only other thing I would do, is possibly add `etc_machine_id` & `insights_client_id` to the test `fact_collection` so that we can check that it shows up in the `expected_insights` dictionary as `rhel_machine_id` and `insights_id` at the top level, and their qpc names in the nested facts (since those two facts are the only ones we have to rename it would be a good regression test).